api/v1/db/oracledb/setupDB.sql

```diff
@@ -0,0 +1,76 @@
+DROP TABLE PARKS;
+DROP TABLE OWNERS;
+
+CREATE TABLE PARKS (
+	ID NUMBER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL,
+	NAME VARCHAR2(225) NOT NULL,
+	STREET_ADDRESS VARCHAR2(225) NOT NULL,
+	CITY VARCHAR2(225) NOT NULL,
+	STATE VARCHAR2(225) NOT NULL,
+	ZIP INTEGER NOT NULL,
+	LATITUDE FLOAT NOT NULL,
+	LONGITUDE FLOAT NOT NULL,
+	OWNER_ID VARCHAR2(225) NOT NULL,
+
+	BALLFIELD BOOLEAN NOT NULL,
+	BBQ BOOLEAN NOT NULL,
+	BASKETBALL BOOLEAN NOT NULL,
+	BIKE_PATHS BOOLEAN NOT NULL,
+	BOAT_RAMPS BOOLEAN NOT NULL,
+	DOGS_ALLOWED BOOLEAN NOT NULL,
+	DRINKING_WATER BOOLEAN NOT NULL,
+	FISHING BOOLEAN NOT NULL,
+	HIKING BOOLEAN NOT NULL,
+	HORSESHOES BOOLEAN NOT NULL,
+	NATURAL_AREA BOOLEAN NOT NULL,
+	DOG_PARK BOOLEAN NOT NULL,
+	FIELDS BOOLEAN NOT NULL,
+	SHELTERS BOOLEAN NOT NULL,
+	TABLES BOOLEAN NOT NULL,
+	PLAY_AREA BOOLEAN NOT NULL,
+	RESTROOMS BOOLEAN NOT NULL,
+	SCENIC_VIEW BOOLEAN NOT NULL,
+	SOCCER BOOLEAN NOT NULL,
+	TENNIS BOOLEAN NOT NULL,
+	VOLLEYBALL BOOLEAN NOT NULL,
```

Comment:
Instead of using multiple `BOOLEAN` columns to represent amenities. You could have a `PARK_AMENITIES` table that has columns for a park id and an amenity. A row in this table would represent that the park has an amenity. To represent an enum of values for the amenities. You could also have an `AMENITIES` table with a row for each amenity.